haskell-library/README.md

```diff
@@ -0,0 +1,22 @@
+Template for building and testing haskell library with multiple versions of GHC via haskell.nix for Buildkite, Gitlab or GitHub CI
+
+## How to use this template
+
+- The recommended way to use this template (if you have nix flakes enabled) is `nix flake init -t github:serokell/templates#haskell-library`
+
+- Otherwise:
+    1. Copy `flake.nix` file to your repository, it contains all nix dependencies and definitions.
+
+    2. Copy `default.nix` and `shell.nix` files which provide compatibility with non-flake nix interface.
+
+    3. Get unstable nix with `nix-command flakes` experimental features enabled, and run `nix flake update` to generate `flake.lock` file. This file pins the latest versions of the nix dependencies.
```

Comment:
> Get unstable nix with `nix-command flakes` experimental features enabled

Looks quite old, flakes are not in unstable nix nowadays (since 2.4), but AFAIR `nix-command flakes`  experimental features are still need to be enabled